src/components/dialogs/network-modifications/static-var-compensator/creation/stand-by-Automaton-form.tsx

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import Grid from '@mui/material/Grid';
+import {
+    ADD_STAND_BY_AUTOMATON,
+    CHARACTERISTICS_CHOICE_AUTOMATON,
+    CHARACTERISTICS_CHOICES,
+    HIGH_VOLTAGE_SET_POINT,
+    HIGH_VOLTAGE_THRESHOLD,
+    LOW_VOLTAGE_SET_POINT,
+    LOW_VOLTAGE_THRESHOLD,
+    MAX_Q_AT_NOMINAL_V,
+    MAX_SUSCEPTANCE,
+    MIN_Q_AT_NOMINAL_V,
+    MIN_SUSCEPTANCE,
+    STAND_BY_AUTOMATON,
+    VOLTAGE_REGULATION_MODE,
+    VOLTAGE_REGULATION_MODES,
+} from 'components/utils/field-constants';
+import { CheckboxInput, FloatInput, SelectInput, SwitchInput, TextInput } from '@gridsuite/commons-ui';
+import { SusceptanceAdornment, VoltageAdornment } from '../../../dialogUtils';
+import { Box } from '@mui/system';
+import React, { useState } from 'react';
+import { useWatch } from 'react-hook-form';
+import { FormattedMessage } from 'react-intl';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Tooltip from '@mui/material/Tooltip';
+import { WarningAmber } from '@mui/icons-material';
+import { SusceptanceArea } from './susceptance-area';
+import { getFloatNumber } from './stand-by-automaton-form-utils';
+
+type FieldKeys =
+    | 'modeAutomaton'
+    | 'standby'
+    | 'lVoltageSetLimit'
+    | 'hVoltageSetLimit'
+    | 'lVoltageThreshold'
+    | 'hVoltageThreshold';
+export const StandbyAutomatonForm = () => {
```

Comment:
file name is not in convention, should rename to stand-by-automaton-form or better with standby-automaton-form
stand-by-automaton-form-utils can be rename as standby-automaton-form-utils